### PR TITLE
followLinkAndCommentsNewTab no longer opens duped tabs.

### PR DESCRIPTION
**Pull request by tersers**
Relevant issue: #4423 
Tested in browser: Chrome

This is a simple stopgap to address #4423, which may have some issues beyond the control of RES. The comments link will now only open once.

### CODE REVIEW

**Comment by mercutiodesign** [comment]
I just tested this code and I noticed two linting errors: 

```
...\lib\modules\singleClick.js
80:16  error  Expected '!==' and instead saw '!='  eqeqeq
85:38  error  Expected '!==' and instead saw '!='  eqeqeq
```

**Comment by tersers** [comment]
Hey there, sorry about that! I'm quite new to JS. The recent commit fixes the linting issues. Thanks for pointing those out!

File: lib/modules/singleClick.js [hunk 1]
```diff
@@ -75,10 +75,16 @@ export function openTabs(thing: Thing, focused: boolean) {
...
-		urls.push(comments);
+		// to avoid opening two of the same link, ensure
+		// the comments link is not the same as the post link.
+		if (comments !== link) {
+			urls.push(comments);
+		}
 	} else if (module.options.openFrontpage.value) {
 		const frontpageLink = thing.getSubredditLink();
-		if (frontpageLink) urls.push(frontpageLink.href);
+		if (frontpageLink && frontpageLink !== link) {
```
Comment (mc10) [inline lib/modules/singleClick.js]: I believe this should be `frontpageLink.href !== link`.

**Comment by erikdesjardins** [comment]
Thanks!

**Pull request merged**